sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -480,7 +480,7 @@ abstract class HiveComparisonTest
                 val executions = queryList.map(new TestHive.QueryExecution(_))
                 executions.foreach(_.toRdd)
                 val tablesGenerated = queryList.zip(executions).flatMap {
-                  case (q, e) => e.sparkPlan.collect {
+                  case (q, e) => e.executedPlan.collect {
```

Comment:
Here we just collect `InsertIntoHiveTable` node to verify its table name is included in `tablesRead`. Next, for each `InsertIntoHiveTable` node, we will call `insert.child.execute().collect()` to run the its child plan.